pkg/providers/redeliver-command/event.go

```diff
@@ -37,7 +37,8 @@ func (c *CommandRedeliver) ProcessIssueCommentEvent(event *github.IssueCommentEv
 	if !strings.Contains(comment, "/run") &&
 		!strings.Contains(comment, "/test") &&
 		!strings.Contains(comment, "/bench") &&
-		!strings.Contains(comment, "/release") {
+		!strings.Contains(comment, "/release") &&
+		!strings.Contains(comment, "/assign") {
```

Comment:
I think `/assign` does not need to be repeated.

Reply:
thanks, I'll fix it.